index.html

```diff
@@ -118,26 +118,79 @@ <h2>Welcome To Fun Bus!</h2>
+                  <p>Pretty simple traveling fun WordPress wanderlust darn simple organized. WordPress travelblogger simple WordPress cute whimsical organized. Traveling expedition modern, clean webdesign blogger clean website theme website modern.</p>
+              </div>
+              <div class="col-sm-6 my-5">
+                  <img src="img/adventure.jpg" class="img-fluid rounded" alt="Let's go on adventure!">
+              </div>
+            </div>
+
+            <div class="row">
+                <div class="col-sm-6 my-5">
+                    <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
+                </div>
+                  <div class="col-sm-6 my-5">
+                    <h4 id="fat">Adventure Awaits</h4>
+                      <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p> 
+                      <p>Pretty simple traveling fun WordPress wanderlust darn simple organized. WordPress travelblogger simple WordPress cute whimsical organized. Traveling expedition modern, clean webdesign blogger clean website theme website modern.</p>
+                  </div>
+              </div>
+
+          <div class="col-12">
+            <h4 id="one">Destinations</h4>
+              <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p> 
+              <p>Pretty simple traveling fun WordPress wanderlust darn simple organized. WordPress travelblogger simple WordPress cute whimsical organized. Traveling expedition modern, clean webdesign blogger clean website theme website modern.</p>
+          </div>
         </div>
+    </div> -->
+    <div class="row">
         <div class="col-sm-6">
-          <h2>Adventure Awaits</h2>
+          <h2>Let's Go!</h2>
           <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p> 
           <p>Pretty simple traveling fun WordPress wanderlust darn simple organized. WordPress travelblogger simple WordPress cute whimsical organized. Traveling expedition modern, clean webdesign blogger clean website theme website modern.</p>
         </div>
-    </div>
+        <div class="col-sm-6">
+            <img src="img/adventure.jpg" class="img-fluid" alt="Let's go on adventure!">
+        </div>
+      </div>
+      <div class="row">
+          <div class="col-sm-6">
+              <img src="img/fun.jpg" class="img-fluid" alt="Lets have fun!">
```

Comment:
It looks like the images above with the `rounded` attribute were commented out. The attribute is missing here; however.